backend/services/implementations/__tests__/classService.test.ts

```diff
@@ -174,8 +176,36 @@ describe("mongo classService", (): void => {
     }).rejects.toThrowError(`Class with id ${notFoundId} not found`);
   });
 
+  it("archive class", async () => {
+    // add test class
+    const classObj = await ClassModel.create(mockTestClass);
+
+    // execute
+    const res = await classService.archiveClass(classObj.id);
+
+    // assert
+    expect(res.isActive).toEqual(false);
+  });
+
+  it("archive class with non-existing id", async () => {
+    const notFoundId = "86cb91bdc3464f14678934cd";
+    await expect(async () => {
+      await classService.archiveClass(notFoundId);
+    }).rejects.toThrowError(`Class with id ${notFoundId} not found`);
+  });
+
+  it("archive class that is already archived", async () => {
+    const classObj = await ClassModel.create(mockTestClass);
+    const archivedClass = await classService.archiveClass(classObj.id);
```

Comment:
rather than making the call twice, i would prefer if you made a new constant or smth like
```suggestion
    const classObj = await ClassModel.create({...mockTestClass, isActive: false});
```
just a best practice, to isolate what we are testing